documentation/book/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='installing-your-own-ca-certificates-{context}']
+= Installing your own CA certificates
+
+This procedure will show you how to install your own CA certificates and private keys instead of using CA certificates and private key generated by the Cluster Operator.
+
+.Prerequisites
+
+* The Cluster Operator up and running
+* An `Kafka` resource within {ProductPlatformName}
+* Your own X.509 certificates and keys in PEM format for the cluster CA or clisters CA. For example, these could be generated by `openssl`, using a command such as:
++
+[source,shell,subs="+quotes"]
+openssl req -x509 -new -days _<validity>_ --nodes -out ca.crt -keyout ca.key
+
+
+.Procedure
+
+. Edit the `Kafka` resource for your cluster, configuring the `Kafka.spec.clusterCa` or `Kafka.spec.clientsCa` object (as appropriate) to _not_ use generated CAs:
```

Comment:
configuring the `Kafka.spec.clusterCa` or `Kafka.spec.clientsCa` object (as appropriate) 

/ >>>>

configuring either the `Kafka.spec.clusterCa` or the `Kafka.spec.clientsCa` object to _not_ use generated CAs: